src/easylink/configuration.py

```diff
@@ -184,8 +184,12 @@ def _get_schema(self, potential_schemas: list[PipelineSchema]) -> PipelineSchema
         Returns
         -------
             The first ``PipelineSchema`` that validates the requested pipeline.
-            If no validated ``PipelineSchema`` is found, `exit()` is called with
-            `errno.EINVAL` and any validation errors are logged.
+
+        Raises
+        ------
+        SystemExit
+            If no requested pipeline is validated, all validation errors found are
```

Comment:
There first part is confusing to me or at least unclear - I'm interpreting this as if any pipeline that is requested does not validate SystemExit will be raised.It looks like this is for multiple pipelines

Reply:
The way we've developed this, it's possible that we might support multiple schemas (currently unclear whether we ever will or not). So what this does is compares the pipeline that the user _wants_ to every supported pipeline (currently only one). If the user pipeline does not validatate schema_1, it moves on to schema_2. If that doesn't validate, it moves on to schema_3, and so on. If it goes through _all_ of the schemas and none of them validated, then easylink exits.

So I think the note is clear but do please lmk if it's still confusing and I'll try and clarify it.